pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.Ericsson</groupId>
+        <artifactId>eiffel-remrem-parent</artifactId>
+        <version>0.0.2</version>
+    </parent>
+    <properties>
+    <eiffel-remrem-publish.version>0.6.2</eiffel-remrem-publish.version>
```

Comment:
I guess <eiffel-remrem-publish.version>0.6.2</eiffel-remrem-publish.version> property declaration not needed , specify version  directly